src/app/preferences/index.tsx

```diff
@@ -0,0 +1,107 @@
+    { section: 'workspace', subsection: 'teams', title: translate('preferences.workspace.teams.title') },
+    { section: 'workspace', subsection: 'billing', title: translate('preferences.workspace.billing.title') },
+    { section: 'account', subsection: 'plans', title: translate('preferences.account.plans.title') },
+    { section: 'account', subsection: 'billing', title: translate('preferences.account.billing.title') },
+    { section: 'account', subsection: 'security', title: translate('preferences.account.security.title') },
+  ];
+
+  const params = new URLSearchParams(window.location.search);
+  const currentSectionParams = params.getAll('section');
+  const currentSubsectionParams = params.getAll('subsection');
+
+  const [activeSection, setActiveSection] = useState<Sections>([{ section: '', subsection: '', title: '' }]);
+
+  useEffect(() => {
+    if (currentSectionParams.length > 0) {
+      const sectionParams = SECTIONS.filter((section) => section.section === currentSectionParams[0]);
+      const subsectionParams = sectionParams.filter(
+        (subsection) => subsection.subsection === currentSubsectionParams[0],
+      );
+      changeSection(sectionParams[0].section, subsectionParams[0].subsection);
+      setHaveParamsChanged(false);
+    } else {
+      dispatch(uiActions.setIsPreferencesDialogOpen(false));
+      setHaveParamsChanged(false);
+    }
+  }, [haveParamsChanged]);
+
+  const changeSection = (onSection, OnSubsection) => {
+    const selectedSection = SECTIONS.filter((section) => section.section === onSection);
+    const selectedSubsection = selectedSection.filter((subsection) => subsection.subsection === OnSubsection);
+    setActiveSection(selectedSubsection);
+  };
+
+  const goSection = (section, subsection) => {
+    navigationService.openPreferencesDialog(section, subsection);
+    changeSection(section, subsection);
+  };
+
+  const onClose = () => {
+    dispatch(uiActions.setIsPreferencesDialogOpen(false));
+    navigationService.closePreferencesDialog();
+  };
+
+  return (
+    <Modal maxWidth="w-full max-w-4xl" className="m-0 flex" isOpen={true} onClose={() => onClose()}>
+      <Helmet>
+        <title>{activeSection[0].title} - Internxt Drive</title>
+      </Helmet>
+
+      {/* SIDEBAR MENU */}
+      <section className="w-56 border-r border-gray-10 px-2.5">
+        <h1 className="py-3 pl-4 text-xl font-semibold">{translate('preferences.title')}</h1>
+        <button
+          className="py-3 pl-4"
+          onClick={() => {
+            goSection('account', 'security');
+          }}
+        >
+          Go Security
+        </button>
+      </section>
+      {/* SIDEBAR MENU */}
+
+      {/* ACTIVE SECTION */}
+      <section className="relative w-full">
+        <button className="absolute right-0 z-50 m-4 flex w-auto" onClick={() => onClose()}>
+          <X size={22} />
+        </button>
+        <h2>{activeSection[0].title}</h2>
+      </section>
```

Comment:
I have checked in the preview and it seems to me that this does not have the same styles as in the design. I also think that the modal has to have a fixed size.

Reply:
For the size of the modal, it is the one specified in the description of the task.
https://inxt.atlassian.net/browse/PB-1810
The styles I have taken from figma, anyway when we integrate the rest of the sections I can check it.